.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "julia.environmentPath": "/Users/elliegibbs/Documents/my_repo/BISC195Labs/src"
```

Comment:
Generally best not to include files like this. You can do `git rm --cached .vscode/*`, and then add it to your `.gitignore` to avoid it happening again